test_suite/test_filenames.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
From personal experience I would name this folder `test` and not `test_suite`. The reason is to run specific tests (or subsets) we might often pass the filepath to pytest in the command line, so having as short a name as possible is handy.